### PR TITLE
Add Yuzhui as admin members

### PR DESCRIPTION
**Pull request by yuzliu**
Related to kubeflow/community-infra#14

### CODE REVIEW

File: kf-kcc-admins.members.txt [hunk 1]
```diff
@@ -8,4 +8,6 @@ google-kubeflow-admins@kubeflow.org
 ricliu@google.com
 rleidle@google.com
 scottleehello@gmail.com
-yangpa@google.com
\ No newline at end of file
+yangpa@google.com
```
Comment (jlewi) [inline kf-kcc-admins.members.txt:11]: Please alphabetize and are these Google accounts?

**Comment by jlewi** [comment]
I think you need to rebase because my PR was merged.

File: kf-kcc-admins.members.txt (hunk 1 above)
Comment (yuzliu) [reply kf-kcc-admins.members.txt:12]: Changed to use my google account.

File: kf-kcc-admins.members.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 derekhh@snap.com
 hejinchi@cn.ibm.com
+liuyuzhui@google.com
```
Comment (jlewi) [inline kf-kcc-admins.members.txt:3]: This doesn't appear to be valid google account.
Do you have an account that you use to access GCP?

Comment (jlewi) [reply kf-kcc-admins.members.txt:3]: You probably meant.
liuyuzhui@gmail.com

Comment (jlewi) [reply kf-kcc-admins.members.txt:3]: I think @yuzisun 's is yuzi.dan@gmail.com

Comment (yuzliu) [reply kf-kcc-admins.members.txt:3]: Added! Thanks!

Comment (jlewi) [reply kf-kcc-admins.members.txt:3]: @yuzliu I think your email is still wrong; otherwise "Welcome to Google" :)

Comment (yuzliu) [reply kf-kcc-admins.members.txt]: @jlewi oops~ haha would love to visit you again after the pandemic.

**Comment by yuzisun** [comment]
@yuzliu Can you rebase this?

**Comment by yuzisun** [comment]
@andreyvelich got us here https://github.com/kubeflow/internal-acls/pull/335, so this can be closed.

**Comment by andreyvelich** [comment]
@yuzisun You should merge this PR also, my change: https://github.com/kubeflow/internal-acls/pull/335 is only for GitHub team to ping corresponding folks, like what @jlewi proposed here: https://github.com/kubeflow/community-infra/issues/14#issuecomment-684839072.

You have to update `kf-kcc-admins.members.txt` to get access to kf-kcc GCP project.

**Comment by yuzisun** [comment]
got it thanks @andreyvelich !

**Comment by yuzliu** [comment]
@yuzisun I rebased and used Bloomberg email for admin members.

**Comment by Bobgy** [comment]
/assign @jlewi 
because I don't have a lot of context here

**Comment by jlewi** [comment]
@yuzliu These need to be GCP accounts. Are these the emails you use to login into GCP? If not can you sign up for GCP accounts with them?

Go to cloud.google.com
Click sign in
Create new account

We would prefer to use corporate accounts as apposed to gmail accounts when granting privileges.

**Comment by jlewi** [comment]
See here:
https://support.google.com/work/android/answer/6371476?hl=en

You need a Google Acount  but you should be able to sign up with any email. So it doesn't have to be a GSuite account or gmail.

**Pull request merged**